### PR TITLE
Wiki: Create Three New Pages & Small Updates to Others

### PR DESCRIPTION
**Pull request by RoboticMind**
* New Wiki Pages
  * About Pools
  * About Forks
  * About UTXOs
  
* Other Changes
   * Add links to those new pages 
   * Add part about the stake limit to the FAQ
   * Mention the testnet cool down time  

Hosted on my fork. Preview of the new pages can be viewed on my fork:
https://roboticmind.github.io/wiki/utxos.html
https://roboticmind.github.io/wiki/forks.html
https://roboticmind.github.io/wiki/pools.html

### CODE REVIEW

File: wiki/faq.md [hunk 1]
```diff
@@ -233,9 +233,25 @@ only.
 ---
 ### What happens if I stake on a fork?
 
-When you stake on a fork, the stake you see is not actually legitimate. When your wallet figures out that it is on a fork, you will see these rewards disappear as the wallet corrects itself. This process happens automatically. While frustrating, once off of the fork, you are still able to stake again. 
+When you stake on a [fork](forks "wikilink"), the stake you see is not actually legitimate. When your wallet figures out that it is on a fork, you will see these rewards disappear as the wallet corrects itself. This process happens automatically. While frustrating, once off of the fork, you are still able to stake again.
```
Comment (barton2526) [inline wiki/faq.md:236]: I would suggest "is not on the main chain"

File: wiki/faq.md [hunk 2]
```diff
@@ -233,9 +233,25 @@ only.
...
 ### What happens if I stake on a fork?
 
-When you stake on a fork, the stake you see is not actually legitimate. When your wallet figures out that it is on a fork, you will see these rewards disappear as the wallet corrects itself. This process happens automatically. While frustrating, once off of the fork, you are still able to stake again. 
+When you stake on a [fork](forks "wikilink"), the stake you see is not actually legitimate. When your wallet figures out that it is on a fork, you will see these rewards disappear as the wallet corrects itself. This process happens automatically. While frustrating, once off of the fork, you are still able to stake again. 
 
 ---
+
+### My pending rewards are stuck at 16384 GRC
+
+If you see your pending rewards at 16384 GRC, you have hit the maximum amount
```
Comment (barton2526) [inline wiki/faq.md:242]:
```suggestion
If you see your pending rewards at 16384 GRC, you have hit the maximum amount of

```

File: wiki/faq.md [hunk 3]
```diff
@@ -233,9 +233,25 @@ only.
...
+Gridcoin that can be sent per stake. When you stake you will only earn 16384 GRC
+and any amount of extra work done will not be included. If you keep hitting this
+limit, make sure you are running your wallet to stake 24/7 (or close to it). If you are, 
+you should get more Gridcoin to stake more frequently. 
+
+If you want to get an estimate of how likely you are to stake *before* that limit, you can try solving 
+for probability using:
+`16384/(magnitude/4) = -ln( 1 - Probability ) * 10000 / Balance * Difficulty`
```
Comment (barton2526) [inline wiki/faq.md:250]: perhaps a note should be added here about what the probability variable _means_

File: wiki/faq.md [hunk 4]
```diff
@@ -233,9 +233,25 @@ only.
...
+
+Note: It is highly unlikely for most users to run into this limit. It requires a very
+high magnitude to possibly run into this
```
Comment (barton2526) [inline wiki/faq.md:253]:
```suggestion
high magnitude to possibly run into this.
```

File: wiki/forks.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
+---
+title: Forks
+layout: wiki
+description: Information about what forks are, the different types of forks any more
```
Comment (barton2526) [inline wiki/forks.md:4]:
```suggestion
description: Information about what forks are, the different types of forks and more
```

File: wiki/forks.md [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+Temporary forks happen by accident, but are automatically resolved by your wallet finding the better chain.
+Hard forks are protocol level changes meaning it cannot be resolved without users
+on the network upgrading their wallet. Soft forks are like a hard fork, but only
+restrict existing rules and don't add their own meaning that existing wallets
+can still get on the right chain, but they can create blocks rejected by the newly updated
+wallets
+
+## Diagram Explanation
+
+![diagram showing a chain of blocks on the top labeled original chain with a split and a second chain at the bottom labeled forked chain ](/assets/img/wiki/fork-diagram.svg){: class="img-fluid w-100"}
```
Comment (barton2526) [inline wiki/forks.md:27]:
```suggestion
![diagram showing a chain of blocks on the top labeled original chain with a split and a second chain at the bottom labeled forked chain](/assets/img/wiki/fork-diagram.svg){: class="img-fluid w-100"}
```

File: wiki/faq.md (hunk 1 above)
Comment (RoboticMind) [reply wiki/faq.md:236]: That wasn't part of the change here, but I can do that. Might do "real stake on the main chain" for a similar idea

File: wiki/faq.md (hunk 3 above)
Comment (RoboticMind) [reply wiki/faq.md:251]: It's how likely you are to stake before that limit, so not sure how to define it without repeating myself. Was aiming to use that previous

Comment (RoboticMind) [reply wiki/faq.md:251]: Maybe just "for *that* probability" would work?

Comment (barton2526) [reply wiki/faq.md:250]: I mean that it is a confidence interval.  I would guess most people don't have a concrete understanding of those.

File: wiki/faq.md (hunk 2 above)
Comment (barton2526) [reply wiki/faq.md]: you put "ofs"?

File: wiki/faq.md [hunk 7]
```diff
@@ -526,4 +542,4 @@ it can be impossible to send. To fix this, you can try consolidating them using
 `consolidateunspent` [rpc](rpc "wikilink") command. For each of your addresses,
 try running `consolidateunspent <address>`. After this finishes, try sending
 the transaction again. This may require multiple rounds of the command if you
-have a lot of small transactions
\ No newline at end of file
+have a lot of small transactions
```
Comment (barton2526) [inline wiki/faq.md:545]:
```suggestion
have a lot of small transactions.

```

**Comment by barton2526** [comment]
nit: names like Alice and Bob should probably be capitalized

File: wiki/utxos.md [hunk 8]
```diff
@@ -0,0 +1,197 @@
...
+You would use up that 50 Gridcoin UTXO to in a transaction as an input and have 
+two outputs: one to alice with the 25 Gridcoin and another to yourself 
+with the leftover 25 Gridcoin (you must use up all of a UTXO). 
+
+You went from having 1 UTXO that was 50 Gridcoin large to having 1 new UTXO that
+is 25 Gridcoin large. The 50 Gridcoin output is now spent and cannot be used again
+
+
+### Example 2
+Suppose you wanted to send your 400 Gridcoin of your balance of 500 Gridcoin
```
Comment (barton2526) [inline wiki/utxos.md:46]:
```suggestion
Suppose you wanted to send 400 Gridcoin of your balance of 500 Gridcoin 
```

File: wiki/utxos.md [hunk 9]
```diff
@@ -0,0 +1,197 @@
...
+
+## Relation to Cooldown & Efficiency
+
+When a new UTXO is created, it is unable to stake for the first 16 hours[^3].
+This time is called a cooldown. The aim of the cooldown is to make a 51% 
+attack more difficult
+
+Since staking creates a new UTXO, that UTXO goes on cooldown. This means that
+when you stake, part of your balance will be offline and unable to stake. Thus
+making your balance split across smaller UTXOs will make staking more efficient
```
Comment (barton2526) [inline wiki/utxos.md:161]:
```suggestion
having your balance split across smaller UTXOs will make staking more efficient 
```

File: wiki/utxos.md [hunk 10]
```diff
@@ -0,0 +1,197 @@
...
+The reason that the network uses UTXOs instead of storing balances is because
+it is much easier and quicker to lookup and validate a UTXO. Using total balances
+requires looking at every transaction that ever occurred on an address and 
+makes transaction validation much more complex
+
+
+# Other Miscellaneous Notes
+
+* You can control which exact UTXOs are used as inputs and created as outputs
+using the coin control feature in the GUI. If on Ingrid (5.2.1.0) or higher this
```
Comment (barton2526) [inline wiki/utxos.md:180]:
```suggestion
using the coin control feature in the GUI. If on Ingrid (5.3.1.0) or higher this
```

File: wiki/utxos.md [hunk 11]
```diff
@@ -0,0 +1,197 @@
...
+is the dropdown that shows when you send a transaction
+
+
+* The size in bytes (and thus fees) of a transaction is determined largely 
+by the number of inputs and output and not directly with the number of Gridcoin moved
+
+* Trying to use too many UTXOs as either an input or an output in one transaction 
+can make a transaction to be too big in bytes to send. This can be fixed
```
Comment (barton2526) [inline wiki/utxos.md:188]:
```suggestion
can make a transaction too big in bytes to send. This can be fixed
```

File: wiki/faq.md (hunk 2 above)
Comment (RoboticMind) [reply wiki/faq.md]: Whoops. Not even sure how I did that since I just clicked "add suggestion to batch" here

File: wiki/faq.md (hunk 3 above)
Comment (RoboticMind) [reply wiki/faq.md]: Put a little sentence after it that explains it a little bit

**Review by barton2526 — APPROVED** [review]

**Pull request merged**